app/routes/goal_routes.py

```diff
@@ -1 +1,107 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, abort, make_response, request, Response
+
+from app.models.goal import Goal
+from app.models.task import Task
+from ..db import db
+from datetime import datetime
+from app.routes.utilities_routes import *
```

Comment:
Prefer to explicitly list the symbols being imported rather than using `import *`